interfaces/IBF-dashboard/src/app/services/map.service.ts

```diff
@@ -293,21 +293,46 @@ export class MapService {
     waterPoints: GeoJSON.FeatureCollection,
   ) {
     const isLoading = waterPoints ? false : true;
-    this.addLayer({
-      name: IbfLayerName.waterpoints,
-      label: IbfLayerLabel.waterpoints,
-      type: IbfLayerType.point,
-      group: IbfLayerGroup.point,
-      description: this.getPopoverText(layer),
-      active: this.adminLevelService.activeLayerNames.includes(
-        IbfLayerName.waterpoints,
-      ),
-      show: true,
-      data: waterPoints,
-      viewCenter: false,
-      order: 2,
-      isLoading,
-    });
+
+    // HACK: conditionally add waterpoints layer because it has a loading logic (see loadWaterPointsLayer)
+    // which adds the layer to the map asynchronously in portals where it is not configured
```

Comment:
should 'in portals' here specifically be 'in disaster-types'?

Reply:
@jannisvisser I renamed portals to disaster types